### ISSUE TITLE
Refile from current location

### ISSUE THREAD

**Issue by ScottFreeCode** [orgzly/orgzly-android#836]
- [x] I have searched for existing issues that may be the same as or related to mine.

TL;DR from the conclusion below:

> **It would be helpful for the refile dialogue to have a "navigate back to this note's current parent" button…**

I really like the enhancement to remember where you previously refiled to. This is fantastic when using refiling to gather notes into the same place.

Sometimes, however, what I'd like is more or less the opposite: I'd like to refile to somewhere near where I'm refiling from. This can be for archiving in an ARCHIVE sibling (Orgzly could alsp use better support for the standard org mode archive features, but that's another story); but it could also be handy for incremental reorganization: I have for instance a reference tree where the tree as a whole has an "inbox" note where I refile stuff from my Inbox notebook, then the various sections each have an "inbox" note for deciding which subsection it goes in; the structure helps me move a note out of a cluttered inbox without having to commit to deciding the exact final destination just to declutter any one layer. Using refile for this would still be faster than using "move" if I would have to move the note past several other notes, it even could be faster than cut/paste since in my experience refile is a little faster than cut-paste due to the number of actions/gestures required to select, cut, navigate to new location, select, paste, chooses above/below/in… vs. refile: select, refile, navigate using an optimized view, just click the refile icon one more time and boom you're done. (I also don't trust cut/paste because it's not atomic and forgets history: I can easily cut, get interrupted, never remmeber to paste, later cut or copy something else and boom: I've lost the cut note or tree without ever realizing it till it's too late. Tell you what, I'll make sure there are separate issues for atomic cut-paste or viewable/usable copy/cut history.)

In the refile dialogue, I can easily navigate back up the breadcrumb, to the notebook selection or to anywhere up along the hierarchy of wherever the dialogue opens to. **It would be helpful for the refile dialogue to have a "navigate back to this note's current parent" button that would make it equally easy to jump to the nearest notes.** This would also avoid having to change/revert the default behavior of remembering the previous refile destination, which is useful for other cases.
